dogs.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Adopt A Dog - Our Dogs</title>
+  </head>
+
+  <body style="background-color:powderblue;">
+
+<!-- This is the header on the page - in two sections for styling later.  -->
+    <header>
+      <article id="left-header">
+        <a href="index.html"><img src="images/adoptadog-logo.svg" alt="Logo for Adopt A Dog with silouette of dog."></a>
+        <img src="images/header-image.jpg" alt="Background image of brown dog.">
+        <h2 class="header-text">Seattle's premiere dog adoption agency</h2>
+      </article>
+
+      <article id="right-header">
+        <a href="checkout.html"><img src="images/cart-icon.svg" alt="Icon of a shopping cart.">$1234.56</a>
+        <p class="checkout"><a href="checkout.html">Checkout</a></p>
+      </article>
+    </header>
+
+<!--This is the navigation - current page has id of active for styling in navigation later-->
+    <ul>
+      <li class="navigation"><a href="index.html">Home /</a></li>
```

Comment:
Seeing as you're on a great start and your code looks awesome, see if you can figure out with CSS how to inject the / marks. It's totally extra credit, but a nice touch

Reply:
the dividign / marks....just to be clear

Reply:
Shall do.